scaladoc/resources/dotty_res/scripts/staticsite/main.js

```diff
@@ -0,0 +1,77 @@
+document.addEventListener("DOMContentLoaded", function () {
+  const tabContainers = document.querySelectorAll('.tabs');
+  tabContainers.forEach(container => {
+    const radios = container.querySelectorAll('.tab-radio');
+    const labels = container.querySelectorAll('.tab-label');
+    const contents = container.querySelectorAll('.tab-content');
+
+    // Hide all tab contents except the first
+    contents.forEach((content, index) => {
+      if (index !== 0) content.style.display = 'none';
+    });
+
+    // Check the first radio button
+    if (radios.length > 0) radios[0].checked = true;
+    contents[0].style.display = 'block'; // Ensure the first tab content is displayed
+
+    labels.forEach((label, index) => {
+      label.addEventListener('click', () => {
+        // Hide all tab contents
+        contents.forEach(content => content.style.display = 'none');
+
+        // Show the clicked tab's content
+        contents[index].style.display = 'block';
+      });
+    });
+  });
+});
```

Comment:
there's a bug here in that by default all tabs are hidden, and only the last tab button works, but then it shows the contents of the first tab - https://new-docs-scala-lang.vercel.app/docs/getting-started/index.html

Reply:
in fact there is an exception on line 15 (`undefined is not an object, evaluating contents[0].style`)

Reply:
however this bug seems to appear in Safari, but not Chrome